include/yokozuna.hrl

```diff
@@ -263,6 +263,12 @@
 -define(YZ_ENTROPY_LOCK_TIMEOUT,
     app_helper:get_env(?YZ_APP_NAME, anti_entropy_lock_timeout, 10000)).
 
+-define(YZ_ENTROPY_THROTTLE_KEY, aae_throttle).
+-define(YZ_ENTROPY_THROTTLE_LIMITS_KEY, aae_throttle_limits).
+-define(YZ_ENTROPY_THROTTLE_DEFAULT_LIMITS,
+        [{-1,0}, {500,10}, {1000,50}, {5000,250}, {10000,1000}, {50000,3000}]).
```

Comment:
I want to point out that these numbers are complete guesses. We need to do some testing and tuning to make sure that they are effective.